src/Chain.hpp

```diff
@@ -25,11 +26,11 @@ class Chain : public Smooshish {
  private:
   SmooshishPtr prev_;
   SmooshablePtr curr_;
-  int left_flex_ = 0;
-  int right_flex_ = 0;
+  int left_flex_;
+  int right_flex_;
```

Comment:
Another semantics thing.  As I read this, I wondered why we initialized flexes to 0 (instead of default-construct), when the Chain constructor already initializes values for `left_flex_` and `right_flex_`? (i.e. there'll never be a Chain that has those explicit 0's for flexes b/c they'll be overwritten in the constructor?)